### PR TITLE
Smbus

### PR DESCRIPTION
**Pull request by alanch-ms**
- Added winrt_smbus library
- Add 'site-packages' to the default python path
- Fix a bad project reference

### CODE REVIEW

**Comment by alanch-ms** [comment]
@munyirik , @alecont - Please CR.

File: extensions/winrt_smbus/winrt_smbus.py [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+
+class SMBus:
+    """
+    SMBus([bus]) -> SMBus
+
+    Return a new SMBus object that is (optionally) connected to the
+    specified I2C device interface.
+    """
+
+    combine_write_read = False
```
Comment (munyirik) [inline extensions/winrt_smbus/winrt_smbus.py:11]: why do you mention raspbian?

File: extensions/winrt_smbus/winrt_smbus.py [hunk 2]
```diff
@@ -0,0 +1,264 @@
+import _wini2c as i2c
+
+class SMBus:
```
Comment (munyirik) [inline extensions/winrt_smbus/winrt_smbus.py:3]: I'm not really familiar with SMBus. Looks like you're just using i2c. Are they the same?

Comment (alanch-ms) [inline extensions/winrt_smbus/winrt_smbus.py:3]: Here's the spec for smbus - http://www.smbus.org/specs/.  They are similar, but not same.  It's more like SMBus is built on top of i2c.

File: extensions/winrt_smbus/winrt_smbus.py (hunk 1 above)
Comment (alanch-ms) [inline extensions/winrt_smbus/winrt_smbus.py:11]: B/c we are emulating raspbian behavior by default.

**Comment by alecont** [comment]
looks good. merging.

**Pull request merged**